platform/mac/mac.gradle

```diff
@@ -11,16 +11,25 @@ tasks.register('exportMac64', Copy.class) {
     dependsOn jar
     dependsOn exportPlugins
 
+    def icon = "mcreator.icns"
+    def appIcon = "mcreatorapp.icns"
+    if (((String) project.mcreatorconf.getProperty('mcreator')).contains("EAP")) {

```

Comment:
Please make solution that always uses icon.png and renames the file instead as it is cleaner.

Renaming the file is not a problem in Gradle

Reply:
???? macOS doesn't use `icon.png`... it uses both `mcreator.icns` and `mcreatorapp.icns`!

Reply:
You can probably understand I meant mcreator.icns.

My point still was to make a solution with file renames, not changed references to different file names.

I would also like to kindly ask you to avoid passive-aggressive comments withing the community as they are not welcomed.

Reply:
Also arguing about every single remark me or any other contributor makes will result in your PRs being on hold for longer and divert interest from people reviewing them

Reply:
Still don't get your point

Reply:
It's done, I did what you said...